pyrate_limiter/bucket.py

```diff
@@ -178,6 +186,7 @@ def put(self, item: float):
 
         if current_size < self.maxsize():
             conn.rpush(self._bucket_name, item)
+            conn.expire(self._bucket_name, self.maxinterval())
```

Comment:
Just one quick question: is it ok to expire with maxinterval=0?

Reply:
Hi, I tested with 0 and it's okay. Doesn't produce any error and simply expires instantly. Initially I went with -1 for unlimited persistence, but 0 seems to be better to catch any mistakes early on similar to the maxsize parameter. At least that's my thought process :)

Reply:
Expiring instantly is not really ok - i mean if our fellow developers bump the version, they would not know about this default behavior when their keys instantly disappear unnoticed. That would be what we usually call `incompatibility` isn it? 

I think we can put None as the default value and only call the expire action if maxinterval is explicitly set.

Reply:
And besides, what we are talking about here is the time to live of a key in memory, while in the code we already refer `interval` as the time between expiration of values. 

I feel the `maxinterval` term is a bit specific to redis bucket. So I think it's better to stay within the Redis bucket implementation. The abstract bucket can remain the same.

Reply:
Agreed, it wouldn't be fair to existing users who depend on unlimited persistence. May I ask why we can't simply use `-1` as default value since it's same as the current behavior, i.e., unlimited persistence. Anyway, your call on that!

Reply:
If we only change the implementation of redis-bucket, the -1 is generally ok, but it will cost us another call to redis server. I think we can save that bit of execution.

Reply:
@vutran1710 I made the discussed changes, could you have a look? 😉

Reply:
PR approved, 
but the test has not been able to run (dont know why tho)
plus, I have some additional thoughts on this so we will wait for a few days.

Reply:
Don't know why the tests failed to complete here, but I checked out your PR and updated with some changes here #82 (mostly wording & tests) and tests now seem alright.
Feel free to comment there while I'm going to close this PR.
Thank you for the contribution.